tests/test_geometry.py

```diff
@@ -18,6 +19,26 @@ def test_pickleable():
     assert poly == unpickled
 
 
+def test_geobox_simple():
+    from affine import Affine
+    t = geometry.GeoBox(4000, 4000,
+                        Affine(0.00025, 0.0, 151.0, 0.0, -0.00025, -29.0),
+                        geometry.CRS('EPSG:4326'))
+
+    expect_lon = np.asarray([151.000125,  151.000375,  151.000625,  151.000875,  151.001125,
+                             151.001375,  151.001625,  151.001875,  151.002125,  151.002375])
+
+    expect_lat = np.asarray([-29.000125, -29.000375, -29.000625, -29.000875, -29.001125,
+                             -29.001375, -29.001625, -29.001875, -29.002125, -29.002375])
+    expect_resolution = np.asarray([-0.00025, 0.00025])
+
+    assert (np.abs(np.r_[t.resolution] - expect_resolution) < 1e-6).all()
+    assert t.coordinates['latitude'].values.shape == (4000,)
+    assert t.coordinates['longitude'].values.shape == (4000,)
+    assert (np.abs(t.coords['latitude'].values[:10] - expect_lat) < 1e-6).all()
+    assert (np.abs(t.coords['longitude'].values[:10] - expect_lon) < 1e-6).all()
```

Comment:
I have a soft preference for writing floating point comparisons using helpers, provided by either `numpy` or `pytest`.

eg.
```python
from pytest import approx

assert t.coords['latitude'].values[:10] == approx(expect_lat)
assert t.coords['longitude'].values[:10] == approx(expect_lon)
```

Reply:
Agree, I found out about `numpy.testing.assert_almost_equal` after I done pull request, which do we prefer/use already? I'm on the fence about `approx` vs `numpy`, I like `approx` because `assert` is used explicitly, but then it's a bit magical.